lecture_1/cli_args.ts

```diff
@@ -0,0 +1,31 @@
+const {stat} = require("fs/promises");
+
+const args = process.argv.slice(2);
+
+const filePath = args[0];
```

Comment:
Please read the requirements carefully. Path should be provided using --path flag.

lecture_1/hw1.md